examples/with-typescript/pages/index.tsx

```diff
@@ -1,8 +1,8 @@
-import * as React from 'react'
+import Link from 'next/link'
 import MyComponent from '../components/MyComponent'
-
-export default () => 
+export default () =>
   <div>
-    <h1>Hello world</h1>
+    <p>Hello there</p>
     <MyComponent />
-  </div>
+    <Link prefetch href='/pagewithasync'><a>Reactpage</a></Link>
```

Comment:
are you really supposed to put an anchor tag inside a `Link`? seems like the `Link` _is_ the anchor tag.